batch.py

```diff
@@ -357,10 +831,97 @@ def manage_concurrency_for_job(job: hb.job.Job):
             window_size=window_size,
             plink_file=plink_file,
         )
+        genotype_jobs.append(plink_job)
+
+    # the next phase will be done for each cell type
+    celltype_list = celltypes.split(' ')
```

Comment:
This pre-processing might make more sense to go at the start of the function

Reply:
done